pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -112,45 +110,24 @@ func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
 
 // NewStorage creates a new rest.Storage responsible for accessing ServiceBinding
 // resources
-func NewStorage(opts server.Options) (rest.Storage, rest.Storage, error) {
-	prefix := "/" + opts.ResourcePrefix()
-
-	storageInterface, dFunc := opts.GetStorage(
-		&servicecatalog.ServiceBinding{},
-		prefix,
-		bindingRESTStrategies,
-		NewList,
-		nil,
-		storage.NoTriggerPublisher,
-	)
+func NewStorage(optsGetter generic.RESTOptionsGetter) (serviceBindings, serviceBindingStatus rest.Storage, err error) {
+	strategy := bindingRESTStrategies
```

Comment:
I don't see any reason to do these local assignments.

Reply:
Removed.